docs/source/tutorials/sveltekit.mdx

```diff
@@ -0,0 +1,431 @@
+      fallback: null,
+      precompress: false,
+      strict: true,
+    }),
+  },
+  preprocess: vitePreprocess(),
+};
+export default config;
+```
+
+Set `prerender` option to your root layout, `src/routes/+layout.js`:
+
+```js
+export const prerender = true;
+```
+
+### Step 3: Install and configure Transformers.js
+
+You can install Transformers.js from [NPM](https://www.npmjs.com/package/@xenova/transformers) with the following command:
+
+```bash
+npm i @xenova/transformers
+```
+
+Next, we'll create a new [Web Worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers) script where we'll place all ML-related code.
+This is to ensure that the main thread is not blocked while the model is loading and performing inference.
+For this application, we'll be using [`Xenova/distilbert-base-uncased-finetuned-sst-2-english`](https://huggingface.co/Xenova/distilbert-base-uncased-finetuned-sst-2-english),
+a ~67M parameter model finetuned on the [Stanford Sentiment Treebank](https://huggingface.co/datasets/sst) dataset. Add the following code to `./src/lib/worker.js`:
+
+```js
+import { pipeline, env } from "@xenova/transformers";
+
+// Skip local model check
+env.allowLocalModels = false;
+
+// Use the Singleton pattern to enable lazy construction of the pipeline.
+class PipelineSingleton {
+  static task = "text-classification";
+  static model = "Xenova/distilbert-base-uncased-finetuned-sst-2-english";
+  static instance = null;
+
+  static async getInstance(progress_callback = null) {
+    if (this.instance === null) {
+      this.instance = pipeline(this.task, this.model, { progress_callback });
+    }
+    return this.instance;
+  }
+}
+
+// Listen for messages from the main thread
+self.addEventListener("message", async (event) => {
+  // Retrieve the classification pipeline. When called for the first time,
+  // this will load the pipeline and save it for future use.
+  let classifier = await PipelineSingleton.getInstance((x) => {
+    // We also add a progress callback to the pipeline so that we can
+    // track model loading.
+    self.postMessage(x);
+  });
+
+  // Actually perform the classification
+  let output = await classifier(event.data.text);
+
+  // Send the output back to the main thread
+  self.postMessage({
+    status: "complete",
+    output: output,
+  });
+});
+```
+
+### Step 4: Design the user interface
+
+We'll define the main interface and logic for our application on the single page component `./src/routes/+page.svelte` that maps to `/`.
+Learn more how SvelteKit handles [routing](https://kit.svelte.dev/docs/routing).
+
+Let's started by loading the Web Worker as soon as the page is mounted. On Svelte, we can use the `onMount` function to run code when the component is mounted.
+The worker code is referenced via the alias `$lib/worker.js`, since SvelteKit is build with Vite.js, we can use its special syntax `?worker` to indicate that this file should be loaded as a Web Worker.
```

Comment:
```suggestion
The worker code is referenced via the alias `$lib/worker.js`. Since SvelteKit is build with Vite.js, we can use the special syntax `?worker` to indicate that this file should be loaded as a Web Worker.
```